src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -216,6 +217,28 @@ public void Send_SpecifiedPickupDirectory()
             Assert.Equal(".eml", Path.GetExtension(files[0]));
         }
 
+        [Theory]
+        [MemberData(nameof(MessageBodyData))]
+        public void Send_SpecifiedPickupDirectory_MessageBodyDoesNotEncodeForTransport(string messageBody)
+        {
+            Smtp.DeliveryMethod = SmtpDeliveryMethod.SpecifiedPickupDirectory;
+            Smtp.PickupDirectoryLocation = TempFolder;
+            Smtp.Send("mono@novell.com", "everyone@novell.com", "introduction", messageBody);
+
+            string[] files = Directory.GetFiles(TempFolder, "*");
+            Assert.Equal(1, files.Length);
+            Assert.Equal(".eml", Path.GetExtension(files[0]));
+
+            string message = File.ReadAllText(files[0]).Replace("=\r\n","");  // strip out folding
+            Assert.EndsWith($"{messageBody}\r\n", message);
+        }
+
+        public static IEnumerable<object[]> MessageBodyData()
+        {
+            for (int i = 0; i < 100; i++)
+                yield return new object[] { new string('a', i) + '.' };
```

Comment:
Do we really need all 100 of these tests?

Reply:
Since the error only shows up when the period happens to land on a new line, I think we should bracket the max line length. This protects us from a change rendering the test ineffective, say in the case that max length is determined to have an off by one error or something. However, 100 is more than we need. I’ll narrow the range.

Reply:
Can we instead add one test that validates the line length is what we expect?  It can include a comment that indicates if it ever fails because the line length changed, this test will need to be updated to.  Then this can shrink to just one or two test inputs.

Reply:
Done. Verified test fails pre-fix and passes post-fix.